.github/workflows/preprocess-deploy.yml

```diff
@@ -32,12 +32,25 @@ jobs:
     strategy:
       matrix:
         arch: [ARM, ARM_HYP, AARCH64, RISCV64, X64]
-        # no MCS here, auto-updating mcs.xml should be a separate job.
+        # no MCS here, see separate preprocess-mcs.
     steps:
     - uses: seL4/ci-actions/preprocess@master
       with:
         L4V_ARCH: ${{ matrix.arch }}
 
+  preprocess-mcs:
+    name: Preprocess (MCS)
+    needs: code
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [RISCV64]
```

Comment:
Is it intentional to run this only for RSICV64? Don't we have a 32-bit arch that has verified MCS?

Reply:
Yes, C verification is currently only being done for RISCV64.

When that is finished, the plan is to extend, but we'll need to see how fast that goes. The hope is relatively fast, because MCS mostly doesn't touch architecture specific stuff, but it will still be a lot of work. Until then we don't want to block on other architectures.